geocity/apps/submissions/views.py

```diff
@@ -1141,6 +1134,25 @@ def submission_select_administrative_entity(request, submission_id=None):
         else None
     )
 
+    quick_access_slug = request.GET.get("q")
+    if not submission and quick_access_slug:
+        form = get_selectable_form(
+            user=request.user, quick_access_slug=quick_access_slug
+        )
+
+        if form:
+            entities = form.administrative_entities.all()
+
+            if len(entities) == 1:
+                submission = models.Submission.objects.create(
+                    administrative_entity=entities[0],
+                    author=request.user,
+                )
+                submission.forms.set([form])
```

Comment:
If the same slug is defined on multiple forms of the same entity, multiple forms can be selected with a slug, is that correct and wished @rbovard ?

Reply:
Just in case, since slugs are stored in a `unique` column of the Form table, the "if" you mention here cannot be realized: only one form may have a given slug at a time.